Description: Install swell-foop setgid so it can write scores files
Author: Michael Biebl <biebl@debian.org>
Bug: https://bugzilla.gnome.org/show_bug.cgi?id=676456
Index: gnome-games-3.4.2/swell-foop/src/Makefile.am
===================================================================
--- gnome-games-3.4.2.orig/swell-foop/src/Makefile.am	2012-05-15 06:37:50.000000000 +0200
+++ gnome-games-3.4.2/swell-foop/src/Makefile.am	2012-05-21 03:55:08.259007508 +0200
@@ -32,4 +32,9 @@
 	$(CLUTTER_GTK_LIBS)	\
 	$(INTLLIBS)
 
+install-exec-hook:
+	-if test "$(setgid)" = "true"; then \
+	  chgrp $(scores_group) $(DESTDIR)$(bindir)/swell-foop && chmod 2555 $(DESTDIR)$(bindir)/swell-foop ;\
+	fi
+
 -include $(top_srcdir)/git.mk
